src/input_cp2k_mm.F

```diff
@@ -1481,7 +1481,8 @@ SUBROUTINE create_NEQUIP_section(section)
 
       CALL keyword_create(keyword, __LOCATION__, name="ATOMS", &
                           description="Defines the atomic kinds involved in the NEQUIP potential. "// &
-                          "Provide a list of each element.", &
+                          "Provide a list of each element ordered by atomic number. "// &
```

Comment:
What happens when they are not ordered? Does it abort with a nice error message or simply return wrong results?